src/thunderbots/software/ai/primitive/catch_primitive.cpp

```diff
@@ -0,0 +1,43 @@
+#include "ai/primitive/catch_primative.h"
+
+const std::string CatchPrimitive::PRIMITIVE_NAME = "Catch Primitive";
+
+CatchPrimitive::CatchPrimitive(unsigned int robot_id, double velocity,
+                               double dribbler_speed,  double margin)
+                               : robot_id(robot_id)
+                               , velocity(velocity)
```

Comment:
nit: would be preferable if commas were at the end of the line

Reply:
I agree with Henry.

Reply:
Not sure about this, but this style might be enforced by clang-format. Just a heads up. (If it is, we can open a separate PR to deal with it)

Reply:
Ah, just noticed this failed clang-format. @matthewberends can you run fix_formatting.sh from the clang_format folder?